src/asm/rgbasm.5

```diff
@@ -0,0 +1,279 @@
+Valid banks are 0 and 1 but bank 1 isn't available if DMG mode is enabled in
+.Xr rgblink 1 .
+.It Sy SRAM
+A banked external (save) RAM section.
+Mapped to memory at $A000–$BFFF.
+Can only allocate memory, not fill it.
+Valid banks range from 0 to 15.
+.It Sy WRAM0
+A general-purpose RAM section.
+Mapped to memory at $C000–$CFFF, or $C000-$DFFF if DMG mode is enabled in
+.Xr rgblink 1.
+Can only allocate memory, not fill it.
+.It Sy WRAMX
+A banked general-purpose RAM section.
+Mapped to memory at $D000–$DFFF.
+Can only allocate memory, not fill it.
+Valid banks range from 1 to 7.
+Not available if DMG mode is enabled in
+.Xr rgblink 1.
+.It Sy OAM
+An object attributes RAM section.
+Mapped to memory at $FE00-$FE9F.
+Can only allocate memory, not fill it.
+.It Sy HRAM
+A high RAM section.
+Mapped to memory at $FF80–$FFFE.
+Can only allocate memory, not fill it.
+.Pp
+NOTE: If you use this method of allocating HRAM the assembler will NOT choose
+the short addressing mode in the LD instructions
+.Sy LD [$FF00+n8],A No and Sy LD A,[$FF00+n8] No because the actual address
+calculation is done by the linker.
+If you find this undesirable you can use
+.Ar RSSET No / Ar RB No / Ar RW No instead or use the
+.Sy LDH [$FF00+n8],A No and Sy LDH A,[$FF00+n8] No syntax instead.
+This forces the assembler to emmit the correct instruction and the linker to
+check if the value is in the correct range.
+.El
+.Pp
+A section is usually defined as a floating one, but the code can restrict where
+the linker can place it.
+.Pp
+If a section is defined with no indications, it is a floating section.
+The linker will decide where to place it in the final binary and it has no
+obligation to follow any specific rules.
+The following example defines a section that can be placed anywhere in any ROMX
+bank:
+.Pp
+.Dl SECTION \*qCoolStuff\*q,ROMX
+.Pp
+If it is needed, the following syntax can be used to fix the base address of the
+section:
+.Pp
+.Dl SECTION \*qCoolStuff\*q,ROMX[$4567]
+.Pp
+It won't, however, fix the bank number, which is left to the linker.
+If you also want to specify the bank you can do:
+.Pp
+.Dl SECTION \*qCoolStuff\*q,ROMX[$4567],BANK[3]
+.Pp
+And if you only want to force the section into a certain bank, and not it's
+position within the bank, that's also possible:
+.Pp
+.Dl SECTION \*qCoolStuff\*q,ROMX,BANK[7]
+.Pp
+In addition, you can specify byte alignment for a section.
+This ensures that the section starts at a memory address where the given number
+of least-significant bits are 0.
+This can be used along with
+.Ar BANK No , if desired.
+However, if an alignment is specified, the base address must be left unassigned.
+This can be useful when using DMA to copy data or when it is needed to align the
+start of an array to 256 bytes to optimize the code that accesses it.
+.Pp
+.Dl SECTION \*qCoolStuff\*q,ROMX,ALIGN[8] ; align to 256 bytes
+.Pp
+.Dl SECTION \*qCoolStuff\*q,ROMX,BANK[2],ALIGN[4] ; align to 16 bytes
```

Comment:
That was because I copy pasted the first one because of the weird way to add double quotes and didn't realise they were different.

I'll change the one of 256 bytes back to WRAM0, it makes more sense.